lib/countly-common.js

```diff
@@ -75,17 +75,43 @@ var cc = {
     },
     /**
      *  Log data if debug mode is enabled
+     * @param {string} level - log level (error, warning, info, debug, verbose)
+     * @param {string} message - any string message
+     * @param {boolean} debug - debugging is on if true
      */
-    log: function log() {
+    log: function log(level, message, ...args) {
         if ((this.debug || this.debugBulk || this.debugBulkUser) && typeof console !== "undefined") {
-            if (arguments[1] && typeof arguments[1] === "object") {
-                arguments[1] = JSON.stringify(arguments[1]);
+            if (args[0] && typeof args[0] === "object") {
+                args[0] = JSON.stringify(args[0]);
             }
+            var color = '';
+            //change color
+            if (level === this.logLevelEnums.ERROR) {
+                //red color
+                color = '\x1b[31m%s\x1b[0m';
+            }
+            else if (level === this.logLevelEnums.WARNING) {
+                //yellow
+                color = '\x1b[33m%s\x1b[0m';
+            }
+            else if (level === this.logLevelEnums.DEBUG) {
+                //cyan
+                color = '\x1b[36m%s\x1b[0m';
+            }
+            else if (level === this.logLevelEnums.VERBOSE) {
+                //blue
+                color = '\x1b[34m%s\x1b[0m';
+            }
+            else {
+                //default log level is info
+                level = this.logLevelEnums.INFO;
+            }
+
             // eslint-disable-next-line no-console
-            console.log(Array.prototype.slice.call(arguments).join("\n"));
+            console.log(color, level + message, Array.prototype.slice.call(args).join("\n"));
```

Comment:
depending on the log level, you should use the available "logWarning" and "logError" functions